01-JS-DOM-APIs/code.js

```diff
@@ -0,0 +1,104 @@
+function show(){
+                let obj = document.getElementById("hidden");
+                if(obj.style.display == "block")
+                        obj.style.display = "none";
+                else
+                        obj.style.display = "block";
+                      }
+
+function eventRequest() {
+                        let xhttp = new XMLHttpRequest();
+                        xhttp.open("GET", "http://api.icndb.com/jokes/random", true);
+                        xhttp.onreadystatechange = function(event) {
+                          let response = JSON.parse(event.target.response);
+                          document.getElementById("hidden").innerHTML = response.value.joke;
+                           if (xhttp.readyState === XMLHttpRequest.DONE) {
+                             if (xhttp.responseText.value) {
+                                 document.getElementById("buttonSection").innerHTML  = xhttp.responseText.value.joke;
+                             }
+                           }
+                       }
+                      xhttp.send();
+                    }
+/*Ejercicio 7*/
+function eventOnClick(){
+  let config = {
+    url: 'http://api.icndb.com/jokes/random'
```

Comment:
the method request is also a parameter that you should make configurable, as you may want to reuse that function to make get, post, del requests, etc